From: Darik Horn <dajhorn@vanadac.com>
Date: Tue, 17 Jan 2012 22:23:16 -0600
Subject: Remove userland dist rules.

The `make dist` rule is used to create the DKMS source package, which
does not contain the userland. (eg: Dist --with-config=kernel only.)

For all userland components in each automake template, delete
instances of EXTRA_DIST and change dist_ to nodist_.

Index: pkg-zfs/Makefile.am
===================================================================
--- pkg-zfs.orig/Makefile.am	2014-02-28 12:21:18.929852656 +0100
+++ pkg-zfs/Makefile.am	2014-02-28 12:21:18.945852480 +0100
@@ -20,7 +20,7 @@
 endif
 
 AUTOMAKE_OPTIONS = foreign
-EXTRA_DIST  = autogen.sh copy-builtin
+EXTRA_DIST  =
 EXTRA_DIST += config/config.awk config/rpm.am config/deb.am config/tgz.am
 EXTRA_DIST += META DISCLAIMER COPYRIGHT README.markdown OPENSOLARIS.LICENSE
 
Index: pkg-zfs/cmd/vdev_id/Makefile.am
===================================================================
--- pkg-zfs.orig/cmd/vdev_id/Makefile.am	2014-02-28 12:06:34.629846250 +0100
+++ pkg-zfs/cmd/vdev_id/Makefile.am	2014-02-28 12:21:18.945852480 +0100
@@ -1 +1 @@
-dist_udev_SCRIPTS = vdev_id
+nodist_udev_SCRIPTS = vdev_id
Index: pkg-zfs/etc/zfs/Makefile.am
===================================================================
--- pkg-zfs.orig/etc/zfs/Makefile.am	2014-02-28 12:06:34.645846201 +0100
+++ pkg-zfs/etc/zfs/Makefile.am	2014-02-28 12:21:18.945852480 +0100
@@ -6,4 +6,3 @@
 	vdev_id.conf.sas_switch.example \
 	vdev_id.conf.multipath.example
 
-EXTRA_DIST = $(pkgsysconf_DATA)
Index: pkg-zfs/scripts/zpios-profile/Makefile.am
===================================================================
--- pkg-zfs.orig/scripts/zpios-profile/Makefile.am	2014-02-28 12:06:34.709846204 +0100
+++ pkg-zfs/scripts/zpios-profile/Makefile.am	2014-02-28 12:21:18.945852480 +0100
@@ -1,5 +1,5 @@
 pkgdataprofiledir = $(pkgdatadir)/zpios-profile
-dist_pkgdataprofile_SCRIPTS = \
+nodist_pkgdataprofile_SCRIPTS = \
 	$(top_srcdir)/scripts/zpios-profile/zpios-profile-disk.sh \
 	$(top_srcdir)/scripts/zpios-profile/zpios-profile-pids.sh \
 	$(top_srcdir)/scripts/zpios-profile/zpios-profile-post.sh \
Index: pkg-zfs/scripts/zpios-test/Makefile.am
===================================================================
--- pkg-zfs.orig/scripts/zpios-test/Makefile.am	2014-02-28 12:06:34.709846204 +0100
+++ pkg-zfs/scripts/zpios-test/Makefile.am	2014-02-28 12:21:18.949852443 +0100
@@ -1,5 +1,5 @@
 pkgdatatestdir = $(pkgdatadir)/zpios-test
-dist_pkgdatatest_SCRIPTS = \
+nodist_pkgdatatest_SCRIPTS = \
 	$(top_srcdir)/scripts/zpios-test/16th-8192rc-4rs-1cs-4off.sh \
 	$(top_srcdir)/scripts/zpios-test/1th-16rc-4rs-1cs-4off.sh \
 	$(top_srcdir)/scripts/zpios-test/1x256th-65536rc-4rs-1cs-4off.sh \
Index: pkg-zfs/scripts/zpool-config/Makefile.am
===================================================================
--- pkg-zfs.orig/scripts/zpool-config/Makefile.am	2014-02-28 12:06:34.709846204 +0100
+++ pkg-zfs/scripts/zpool-config/Makefile.am	2014-02-28 12:21:18.949852443 +0100
@@ -1,5 +1,5 @@
 pkgdataconfigdir = $(pkgdatadir)/zpool-config
-dist_pkgdataconfig_SCRIPTS = \
+nodist_pkgdataconfig_SCRIPTS = \
 	$(top_srcdir)/scripts/zpool-config/dm0-raid0.sh \
 	$(top_srcdir)/scripts/zpool-config/file-raid0.sh \
 	$(top_srcdir)/scripts/zpool-config/file-raid10.sh \
